api/v1/db/oracledb/games-dao.js

```diff
@@ -28,7 +28,8 @@ const getGames = async (queries) => {
     SCORE AS "score",
     RELEASE_DATE AS "releaseDate"
     FROM VIDEO_GAMES
-    WHERE ((SCORE BETWEEN :scoreMin AND :scoreMax) OR SCORE IS NULL)
+    WHERE ((SCORE BETWEEN :scoreMin AND :scoreMax)
+    ${sqlParams.scoreMin > 1 || sqlParams.scoreMax < 5 ? '' : 'OR SCORE IS NULL'})
```

Comment:
As discussed, scoreMin and scoreMax should not have any default values so that the null scores game will only show if both of these parameters are not provided.

Reply:
Change scoreMin and scoreMax to not be default and changed the code accordingly.